apps/src/templates/UnsafeRenderedMarkdown.jsx

```diff
@@ -6,9 +6,12 @@ import Parser from '@code-dot-org/redactable-markdown';
 import expandableImages from './plugins/expandableImages';
 import xmlAsTopLevelBlock from './plugins/xmlAsTopLevelBlock';
 import stripStyles from './plugins/stripStyles';
+import externalLinks from 'remark-external-links';
 
 const remarkParser = Parser.create();
-remarkParser.parser.use([xmlAsTopLevelBlock, expandableImages]);
+remarkParser.parser
+  .use([xmlAsTopLevelBlock, expandableImages])
+  .use(externalLinks, {target: '_blank', rel: false});
```

Comment:
we _always_ want to use `rel='noreferrer noopener'` when setting `target=_blank`.

See https://mathiasbynens.github.io/rel-noopener/